### PR TITLE
Flickering lights during voltage puzzle

### LINKED ISSUE #900: Flickering lights script
Create a script that causes the lights in the environment to flicker, giving the impression that something went wrong.

### DoD
The lights will flicker when called by a script.

### PR DESCRIPTION
**Pull request by andens**
When the user provides the wrong fuse configuration during the mastermind puzzle a light will flicker via `VoltageRandomEvents.as` which always causes light flicker for now. This script is a good place to put logic determining what other scripts should be interacted with.

Disclaimer: This code has not been fully tested with VR controls yet.

Resolves #900

### CODE REVIEW

File: Game/Resources/VoltageScene/Scripts/Mastermind.as [hunk 1]
```diff
@@ -86,6 +88,8 @@ class Mastermind {
             if (correct == 4) {
                 puzzleSolved = true;
                 SendMessage(roundabout, 2);
+            } else {
+                SendMessage(randomEventController, 0);
```
Comment (Ittis) [inline Game/Resources/VoltageScene/Scripts/Mastermind.as]: one line statement

File: Game/Resources/VoltageScene/Scripts/VoltageRandomEvents.as [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    VoltageRandomEvents(Entity @entity){
+        @hub = Managers();
+        @self = @entity;
+        @jensLampa = GetEntityByGUID(1511528947);
+    }
+
+    void ReceiveMessage(Entity @sender, int signal) {
+        if (signal == 0) {
+            SendMessage(jensLampa, 0);
```
Comment (Ittis) [inline Game/Resources/VoltageScene/Scripts/VoltageRandomEvents.as]: one line statement

File: Game/Resources/VoltageScene/VoltageScene.json [hunk 3]
```diff
@@ -2774,24 +2774,24 @@
 					"SpotLight" : 
 					{
 						"ambientCoefficient" : 0.0,
-						"attenuation" : 1.0,
+						"attenuation" : 200.0,
```
Comment (Ittis) [inline Game/Resources/VoltageScene/VoltageScene.json]: is this number correct?

File: Game/Resources/VoltageScene/Scripts/Mastermind.as (hunk 1 above)
Comment (Ittis) [reply Game/Resources/VoltageScene/Scripts/Mastermind.as]: :)

File: Game/Resources/VoltageScene/VoltageScene.json (hunk 3 above)
Comment (andens) [reply Game/Resources/VoltageScene/VoltageScene.json]: Yes, a higher value causes light to dissipate more quickly. It's used to make the spotlights beneath the fuses not leak as much behind the electric box.

**Pull request merged**